README.md

```diff
@@ -0,0 +1,16 @@
+# parity-instantseal
+Builds instantseal Docker container, normally available at keydonix/parity-instantseal.
+
+Comes funded with these accounts:
+- 0x913da4198e6be1d5f5e4a40d0667f70c0b5430eb (private key: 0xfae42052f82bed612a724fec3632f325f377120592c75bb78adfcceae6470c5an), loaded with 10^20 ETH
+- These accounts from the mnemonic "zoo zoo zoo zoo zoo zoo zoo zoo zoo zoo zoo wrong", all loaded with 100 ETH
```

Comment:
It appears these are all loaded with 10^20 ETH, not 100 ETH

Reply:
They are loaded with 100 ETH:
```
		"0x913da4198e6be1d5f5e4a40d0667f70c0b5430eb": { "balance": "0xffffffffffffffffffffffffffffffff" },
		"0xfc2077ca7f403cbeca41b1b0f62d91b5ea631b5e": { "balance": "0x56bc75e2d63100000" },

```

Reply:
Misread.  So used to attoeth.